daprdocs/content/en/reference/components-reference/supported-state-stores/setup-inmemory.md

```diff
@@ -0,0 +1,29 @@
+---
+type: docs
+title: "In Memory"
+linkTitle: "In Memory"
+description: "Detailed documentation on the In Memory state component"
+aliases:
+  - "/operations/components/setup-state-store/supported-state-stores/setup-inmemory/"
+---
+
+The In Memory state store component is useful for development purposes and works inside of a single machine boundary.
+
+## Create a Dapr component
+To setup in-memory state store create a component of type `state.in-memory`. See [this guide]({{< ref "howto-get-save-state.md#step-1-setup-a-state-store" >}}) on how to create and apply a state store configuration.
+
+```yaml
+apiVersion: dapr.io/v1alpha1
+kind: Component
+metadata:
+  name: <NAME>
+  namespace: <NAMESPACE>
+spec:
+  type: state.in-memory
+  version: v1
+```
+
```

Comment:
Q: Is there no metadata fields for this? If not, it would still be good to add a heading "Spec metadata fields" and say that there are none, so this is explicit. 

Also I am sure that the schema requires metadata to be there, so it is best to add this into the YAML file above with an empty array.

spec:
  type: state.in-memory
  version: v1
  metadata: []

See this https://github.com/dapr/docs/pull/2527/files#diff-8532dd3adcfbc549dbb03eed88c30ec3988fb81f45bd6f4a775b6e91ed16f144R23

Reply:
Agree with this.